datafusion/sqllogictest/test_files/explain_tree.slt

```diff
@@ -206,14 +206,18 @@ physical_plan
 05)│         FilterExec        │
 06)│    --------------------   │
 07)│         predicate:        │
-08)│string_col@1 != foo AND ...│
-09)└─────────────┬─────────────┘
-10)┌─────────────┴─────────────┐
-11)│      RepartitionExec      │
-12)└─────────────┬─────────────┘
-13)┌─────────────┴─────────────┐
-14)│       DataSourceExec      │
-15)└───────────────────────────┘
+08)│  string_col@1 != foo AND  │
```

Comment:
Can you also add a test for
1.  an even longer expression? One that would take 100 lines or something? I think there should also be a  maximum length limit 
2. a predicate with exactly the render width
3. A predicate with the render width + 1